modules/iam-assumable-role-with-oidc/main.tf

```diff
@@ -1,5 +1,6 @@
 locals {
   aws_account_id = var.aws_account_id != "" ? var.aws_account_id : data.aws_caller_identity.current.account_id
+  provider_url   = "${replace(var.cluster_oidc_url, "https://", "")}"
```

Comment:
1. `var.cluster_oidc_url` - where this is defined?

2. Use Terraform 0.12 short syntax by removing `"${...}"` wrapper.

Reply:
1) No idea, must have been from copying this from our own module. PR is so old I can't remember.
2) Fixed.